### PR TITLE
[READY] Activates CRAB-17 Protocol (New traitor item)

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus (Sprite by PlayerDeer)
add: Adds the Protocol CRAB-17 Phone that can be bought for 6TC and can drain cargo's fund and keep you updated on when cargo makes purchases
/:cl:

A phone that lowers station funds by a value of between 60 to 80%. Can only be used once and is useful for screwing over cargo. but also keeps you updated on when cargo buys or sells things.

https://streamable.com/kup4q

https://streamable.com/z0r8c

### CODE REVIEW

**Comment by optimumtact** [comment]
Okay I definitely chuckled

**Comment by outbools** [comment]
Can I have a quick rundown on this PR?

**Comment by GunHog** [comment]
>Can I have a quick rundown on this PR?
It adds a device that can drain most of Cargo's money on demand.
- Buy "Protocol CRAB-17 Phone"
- (Recommended) Find out how much money cargo has to see if the phone is best used at the moment
- Use phone and confirm the prompt
- Wait a bit and cargo will lost most of its money.

File: code/game/objects/items/miscellaneous.dm
Comment (Mark9013100) [inline code/game/objects/items/miscellaneous.dm]: punctuation

**Comment by Iamgoofball** [comment]
this is good for bitcoin because

**Comment by 81Denton** [comment]
Man, this would be even better if stocks were still around

**Comment by outbools** [comment]
- /tg/ bow to floydanoff
- in contact with catgirls
- possess maintainer-like abilities
- control serb with an iron but fair paw
- own stations & lavalands globally
- Direct descendant of the ancient royal coder line
- Will bankroll the first servers on remake (Floydstation will be the first server)
- Own 99% of var editing research facilities in space
- First designer babies will in all likelihood be floydanoff babies
- said to have 215+ github points, such PR capability on Earth has only existed deep in Github HQ
- Learned fluent DM in under a week
okay its time to stop

**Comment by Militaires** [comment]
image improved sprite, also expect me to push a few PRs to your branch soon
image

**Comment by Militaires** [comment]
actually dont make any further changes to this pr and ill pack the sprite in with some changes you're sure to like

**Comment by davethwave** [comment]
Whoever uses this will be hunted by cargo techs I bet.

**Comment by Therapist1** [comment]
dump it

**Comment by Nickvr628** [comment]
7TC seems high for a one time use item that does not contribute much to your objectives.

Maybe 4 or 5 TC would be more appropriate.

**Comment by Qustinnus** [comment]
Lol misclick

**Comment by Qustinnus** [comment]
But yeah fair I'll make it 5 I suppose, I'm bad at guessing what's a fair amount

**Comment by fizzle7** [comment]
This definitely won't be metagamed

**Comment by 81Denton** [comment]
>This definitely won't be metagamed

Now I can finally metagrudge people who play cargo techs without having to enter their department

**Comment by Qustinnus** [comment]
if you want to metagrudge as traitor you wouldnt have such a difficult time anyways

**Review by Cyberboss — DISMISSED** [review]

**Comment by GravitronBlue** [comment]
Assuming Nuke-ops can buy this I'm rather concerned that this will become a quick way to remotely shut down cargo from ordering auto-rifles and swat gear after declaring war.

Not sure if you could address this now or see how things play out once this is (test)merged.

**Comment by Qustinnus** [comment]
If that does become a problem I'd maybe make it more expensive for nukies.

**Comment by MacHac** [comment]
The problem that I see is that cargo points aren't *that* renewable, and depending on how inconvenient bounties are they can be very labor intensive to get.

I imagine that if hit with this after the low-hanging crates and bounties were sold, the average cargo department would throw its hands up and just go greytide for the rest of the round.  It's what I'd do, especially since these things are cheap enough for it to happen several more times.

I'd argue that (short of an unmanned cargo department), this is as destructive as actively blowing a hole in cargo.  At the very least, its price should reflect that.

**Comment by Qustinnus** [comment]
you can go to lavaland for 5 minutes and be rich. the fact that cargo has little methods of making money is something we should solve seperately. The problem with a lack of funds happens on most roundtypes where things need to be bought whether or not this item exists.

**Comment by 81Denton** [comment]
The whole making money issue aside, you can take a huge dump on a whole department with no way to counter/prevent it.
I think it would generate less salt if it was either a steady drain or revealed the name of the Bogdanoff agent.

**Comment by Qustinnus** [comment]
I might make it a steady drain then.

**Comment by Wilchenx** [comment]
this should also lower value of all stocks to 0 for obvious reasons

**Comment by Kagus** [comment]
> werent stocks removed

Holy shit, this thing is already working, and it hasn't even been merged yet!

---
Considering that this isn't going to have much benefit for a non-nukie (aside from potentially trying to trick the station into believing that it *IS* nukies), I don't know if its lack of a decent counter really means a whole lot, since it's basically only going to get used for trolling anyways and it's a one-off device.

**Comment by Militaires** [comment]
i've made my own pr adding a feature to this https://github.com/Qustinnus/tgstation/pull/2
merge or leave it as you wish

**Comment by optimumtact** [comment]
Cargo defence force in full swing today

**Comment by pizzatiger** [comment]
I don't even play cargo. I just dislike it because it exists solely to ruin someones round with no benefit to you and has no real way to counter or stop that shit.

**Comment by Qustinnus** [comment]
most of the items in the uplink ruin someones round in one way or the other.

**Comment by pizzatiger** [comment]
The main difference is they all have counters/require work. With this you just press a button and it basically can't be stopped/punished unless you were stupid enough to do it in a busy hall and got caught with it

**Comment by Qustinnus** [comment]
https://streamable.com/z0r8c You now get called if something is sold or bought, giving you a good opportunity to see if you should strike.

**Comment by outbools** [comment]
activate merge button

**Comment by fizzle7** [comment]
Add an ATM bug you can remotely control and you must manually install it on the atm. That at least requires some work and skill and can be traced by sec/AI. Although it would make it pretty hard to use now, but hey, not everything has to be ei'nath.

File: code/game/objects/items/miscellaneous.dm
Comment (Militaires) [inline code/game/objects/items/miscellaneous.dm]: >phone stops ringing
"your phone starts ringing" ?????

File: code/game/objects/items/miscellaneous.dm
Comment (Militaires) [inline code/game/objects/items/miscellaneous.dm]: this way, no one else will be able to hear you making the call which blows your cover (which is good gameplay), are you sure you want this strength?

File: code/game/objects/items/miscellaneous.dm
Comment (Militaires) [inline code/game/objects/items/miscellaneous.dm]: bogdanoff*

File: code/game/objects/items/miscellaneous.dm
Comment (Militaires) [reply code/game/objects/items/miscellaneous.dm]: + *bogdanoff

File: code/game/objects/items/miscellaneous.dm
Comment (Qustinnus) [reply code/game/objects/items/miscellaneous.dm]: its a niche item making it overly shit will mean its never used

**Comment by zxaber** [comment]
How about an alternate mode from money siphoning, where the user can force items onto the next purchase order? No limit to the number of items except the limit of Cargo's funds. Purchased items made this way would not show up on the order before the ship is sent, though the little receipt or whatever that comes back might mention an unusual purchase request (not specifying what was actually tacked on). 

The tator would still need to find a way into Cargo to actually collect the goods, though.

**Comment by Qustinnus** [comment]
@Anonmare Github cant play .ogg

**Comment by Enzarel** [comment]
Get this: if you buy it on discount you get a BEAR-BC phone, that instead of making cargo go bankrupt replaces their next purchase with a bear, while still charging them for said purchase

**Comment by pizzatiger** [comment]
@zxaber Thats an amazing idea, It would need a cool down and a part in the code that allows cargo to sell the crates back at full price to stop Nuke ops/traitors from spamming a bunch of pizza crates in an attempt to destroy the cargo budget

**Comment by Qustinnus** [comment]
that makes it completely useless.

**Comment by Shdorsh** [comment]
This asks for a quick reintroduction of stocks [https://www.youtube.com/watch?v=KV5QlSgq7lg](url)

**Comment by ExcessiveUseOfCobblestone** [comment]
So just for clarification, is there any way to thwart this DURING the process of siphoning or do you just have to clean up the mess afterwards?

5 TC to 100% safely sabotage a department at any point in the round is nuts, especially when that item tells you the optimal time to use it to cause the highest damage.

**Comment by Iamgoofball** [comment]
stale close?

**Pull request closed without merging**
